php/php.editor/test/unit/src/org/netbeans/modules/php/editor/completion/PHPCCDocumentationTest.java

```diff
@@ -354,7 +364,21 @@ public void checkCompletionDocumentation(final String file, final String caretLi
     }
 
     private void checkCompletionOnlyDocumentation(String filePath, String caretLine) throws Exception {
-        checkCompletionDocumentation(filePath, caretLine, false, "", QueryType.DOCUMENTATION);
+        checkCompletionOnlyDocumentation(filePath, caretLine, false);
+    }
+
+    private void checkCompletionOnlyDocumentation(String filePath, String caretLine, boolean noDocument) throws Exception {
+        if (!noDocument) {
+            checkCompletionDocumentation(filePath, caretLine, false, "", QueryType.DOCUMENTATION);
+        } else {
+            try {
+                checkCompletionDocumentation(filePath, caretLine, false, "", QueryType.DOCUMENTATION);
+            } catch (AssertionError ex) {
+                // there is no completion item
+                return;
+            }
```

Comment:
https://github.com/apache/netbeans/blob/3e79500397c2c6ca9716049911251fc8e3ba5cdf/ide/csl.api/test/unit/src/org/netbeans/modules/csl/api/test/CslTestBase.java#L3155-L3163

Reply:
I wanted to do that. (I was also worried about that.)
My first idea was to create an empty file if `match` is `null`.
However, "being not shown documentation" is not  "being shown empty documentation", I thought... maybe, although there is no big problem. (Unit tests for the other languages may be affected by that change.)
So, just caught `AssertionError` :) (It seems that `assertNotNull()` are only here)
Do you have any ideas?

Reply:
Hm, I don't know, I am not sure...

@sdedic Sváťo, what do you think? We can keep this PHP change as it is now, of course, no problem.

Thanks.

Reply:
@sdedic Sváťo? Otherwise I will simply merge it, tomorrow. Thanks!